src/Middleware/RateLimiting/src/RateLimiterOptionsExtensions.cs

```diff
@@ -15,11 +15,18 @@ public static class RateLimiterOptionsExtensions
     /// </summary>
     /// <param name="options">The <see cref="RateLimiterOptions"/> to add a limiter to.</param>
     /// <param name="policyName">The name that will be associated with the limiter.</param>
-    /// <param name="tokenBucketRateLimiterOptions">The <see cref="TokenBucketRateLimiterOptions"/> to be used for the limiter.</param>
+    /// <param name="configureOptions">A callback to configure the <see cref="TokenBucketRateLimiterOptions"/> to be used for the limiter.</param>
```

Comment:
Should these say "to be used for the policy."?

Reply:
I chose limiter because the options are used for the limiter that's in the policy, not the policy itself, and since the methods are named `AddXyzLimiter`. The other comments follow this pattern as well